### DISCUSSION TITLE
Help to Install

### DISCUSSION THREAD

**Discussion by Reashle** [merbanan/rtl_433#2812] [Q&A]
Hello

I need help to Install or create this RTL433 on my laptop. I have already a DVBT stick and downloaded the program into Visual Studio, but I don´t know how to install.

Thanks a lot,
BR Markus Wiesinger

**Comment by AutomGuy** [comment]
Hi Markus,
this information is too little. what for a operating system you are use? But first follow these insruction [Building.md](https://github.com/merbanan/rtl_433/blob/master/docs/BUILDING.md)

Best Regards

**Comment by Reashle** [comment]
Hello

My operating system is Windows 11.

Best Regards
________________________________

**Comment by AutomGuy** [comment]
Hi at Windows -> I am out. I have to run my rtl_433 at a linux ubuntu machine.
and that runs without any problems.
BR
